src/viem/prepareTransactions.ts

```diff
@@ -268,7 +268,7 @@ export async function tryEstimateTransactions(
  *
  * @param feeCurrencies
  * @param spendToken
- * @param spendTokenAmount
+ * @param spendTokenAmount BigNumber in smallest unit
```

Comment:
what exactly does smallest unit mean?

Reply:
the smallest unit of the token (e.g., 1ETH => 1e18, 1 usdc => 1e6). I believe we use this term in a few places in the wallet